src/client/components/MealReviews/MealReviews.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./MealReviews.css";
+import { useParams } from "react-router-dom";
+
+const Reviews = (props) => {
+  const { id } = useParams();
+  // console.log(id, props.reviews);
+  const reviewToDisplay =
+    id === undefined
+      ? props.reviews.filter((review) => review.stars > 3)
+      : props.reviews.filter((review) => review.meal_id === Number(id));
+  return (
+    <div className="reviews-details">
+      <ul className="display">
+        {reviewToDisplay.map((review) => (
```

Comment:
Minor detail, but try to keep the naming consistent, it's reviewsToDisplay (plural) right?